content/en/docs/components/katib/hyperparameter.md

```diff
@@ -0,0 +1,436 @@
++++
+title = "Getting started with Katib"
+description = "How to set up Katib and run some hyperparameter tuning examples"
```

Comment:
The fact that the readers would be learning through examples is probably a given, so I'd suggest:

```suggestion
description = "How to set up Katib and perform hyperparameter tuning"
```